frontend/catalyst/pennylane_extensions.py

```diff
@@ -982,6 +984,112 @@ def trace_quantum(self, ctx, device, trace, qrp) -> QRegPromise:
         return qrp2
 
 
+class QCtrl(HybridOp):
+    """Catalyst quantum ctrl operation"""
+
+    def _no_binder(self, *_):
+        raise RuntimeError("QCtrl does not support JAX binding")  # pragma: no cover
```

Comment:
Maybe this could be the default in the base class? (not a must though, just a thought)

Reply:
Done